_opcodes/op116.html

```diff
@@ -10,6 +10,16 @@
 iwd2: 1
 pst: 1
 ---
-Removes invisibility from the targeted creature(s).<br/>
-This effect ignores durations attributed to it.<br/>
+Removes invisibility from the targeted creature(s).
+
+{% capture note %}
+This effect ignores durations attributed to it.
```

Comment:
I added these to prevent programming errors, to avoid a trial and error programming style, setting pseudo-random values until the function does, what they want. The next one who uses their work maybe think they have to set it